templates/zerver/integrations/assembla.md

```diff
@@ -0,0 +1,24 @@
+1. Follow the **Hubot Installation** section of the
+   [Hubot DOC](https://chat.zulip.org/integrations/doc/hubot)
+   to install Hubot.
+
+2. In your Hubot project repo, run:
```

Comment:
Let's point people to https://github.com/hubot-scripts/hubot-assembla, rather than copying the instructions, for two reasons:
* It won't get out of date, in case the instructions at https://github.com/hubot-scripts/hubot-assembla change
* It'll be easier to create a generic template that we can use for all the hubot integrations.